front-end/src/util/util.js

```diff
@@ -0,0 +1,7 @@
+
+// This function trims a long text from beginning to LENGTH and adds ellipses at the end of the stiring
+// length: max length of string, if more than that, it will trim. If not, do nothing.
+// input: the string to trim
+export const trimLongText = (input, length) => {
```

Comment:
Use this to trim the emails as well so that they don't go out of bounds

Reply:
will get worked on different branch